brickflow/codegen/databricks_bundle.py

```diff
@@ -421,15 +422,55 @@ def workflow_obj_to_schedule(workflow: Workflow) -> Optional[JobsSchedule]:
             )
         return None
 
-    def get_entrypoint_notebook_source(self) -> str:
+    def adjust_source(self) -> str:
         return "WORKSPACE" if self.env == BrickflowDefaultEnvs.LOCAL.value else "GIT"
 
-    def task_to_task_obj(self, task: Task) -> Union[JobsTasksNotebookTask]:
+    def adjust_file_path(self, file_path: str) -> str:
```

Comment:
Thanks for creating a common function, we could use this for other task types as well